apps/cli/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@webstudio-is/tsconfig/base.json",
+  "compilerOptions": {
+    "module": "esnext"
```

Comment:
Base already specifies latest. Doesn't it work for you?

Reply:
Yeah, since it runs on  node@18 directly and we need `import-assertions` and ts needs the `module` to be `esnext` to enable it. The base config seems to be `es2020`